src/multi-selector.js

```diff
@@ -122,17 +133,8 @@ const MultiSelector = React.createClass({
 	},
 
 	inputChange: function(newVal) {
-		const filterItems = this.getFilterItems(this.props.items);
-		const activeIndex = this.state.activeIndex;
-
 		this.props.onInputChange && this.props.onInputChange(e.currentTarget.value);
 
-		if (isNull(activeIndex) && filterItems.length !== 0) {
```

Comment:
Turns out this is unnecessary if we just check if `filterItems` has stuff before we try to access the `activeIndex` item of it. (line 166) That means we don't need to get the filter items here